MSAL/src/public/MSALError.h

```diff
@@ -170,6 +170,11 @@ typedef NS_ENUM(NSInteger, MSALErrorCode)
      */
     MSALErrorNoViewController = -42403,
     
+    /*!
+        MSAL tried to open a browser from an extension, which is not allowed.
+     */
+    MSALErrorAttemptToOpenBrowserFromExtension = -42404,
```

Comment:
nit: can we call it something like "openURLFromExtension"?

Reply:
Sure